pkg/haprovider/haprovider.go

```diff
@@ -128,15 +128,15 @@ func (r *HAProvider) createService(
 		r.log.Error(err, "can't unmarshal cluster variables ", "endpoint", endpoint)
 		return nil, err
 	} else if endpoint != "" {
-		// "endpoint" can be ipv4 or hostname, add ipv4 or hostname to service.Spec.LoadBalancerIP
+		// "endpoint" can be ipv4 or hostname, add ipv4 or hostname as annotation: ako.vmware.com/load-balancer-ip:<ip>
 		if net.ParseIP(endpoint) == nil {
 			endpoint, err = QueryFQDN(endpoint)
 			if err != nil {
 				r.log.Error(err, "Failed to resolve control plane endpoint ", "endpoint", endpoint)
 				return nil, err
 			}
 		}
-		service.Spec.LoadBalancerIP = endpoint
+		service.Annotations[akoov1alpha1.AkoPreferredIPAnnotation] = endpoint
```

Comment:
Can we set both annotation and loadbalancerIP for maybe 2 k8s versions. Since customer might use old tkr for new TKG. We should support both for some time